### PR TITLE
Add an option to pipe RDB backups into NodeNorm Redis instead of a full reload

### PR DESCRIPTION
**Pull request by gaurav**
This PR is intended to modify `node-normalization-loader` to add a `restore` mode. If run in `restore` mode, the loader will create a Job for each database, which downloads the RDB file for that database and then [pipes the file into redis-cli](https://stackoverflow.com/a/70022239/27310).

In addition, this incorporates changes from two previous PRs: https://github.com/helxplatform/translator-devops/pull/662 and https://github.com/helxplatform/translator-devops/pull/720, which increase the memory settings for some Redis instances, upgrade Redis to 16.13.2, and add the chemical_drug_db for Drug Chemical conflation.

Things remaining to do:
- [x] Add chemical_drug_db information from ITRB.
- [x] Make sure this is set up so that nothing on ITRB will immediately change -- we can change database information in another PR. We don't want to accidentally break NodeNorm ITRB until we're ready!
- [x] <s>Upgrade NodeNorm to the latest version.</s> Actually, we want to keep this unchanged so we don't break anything on ITRB -- the upgrade will happen in a separate PR (https://github.com/helxplatform/translator-devops/pull/788)

First run timing information:

| Database | Download time (internal at RENCI) | Gunzip time | redis-cli --pipe time | Replies (should be 1 + DBSIZE) |
| --------- | --------------- | ------------ | -------------------- | ------------------------------- |
| chemical-drug-db | 12s | 1s | 1s | 91403 |
| semantic-count | 8s | 0s | 0s | 3315 |
| conflation-db | 1m3s | 18s | 2m12s | 21870951 |
| id-id | 2m21s | 3m3s | 51m30s | 573200982 |
| id-eq-id | 5m25s | 7m48s | 50m40s | 428783557 |
| id-categories | 1m33s | 1m58s | 1h2m48s | 428783557 |
| info-content | 0s | 0s | 23s | 3054853 |

### CODE REVIEW

**Comment by gaurav** [comment]
@YaphetKG I've sent this to you for review because it's a big PR that should have all the major changes in it, but please don't merge it in yet -- I still need to get chemical_drug_db information from ITRB and double-check that, once merged, this won't mess up the current NodeNorm instances on ITRB.

**Review by YaphetKG — APPROVED** [review]

**Pull request merged**
